### ISSUE TITLE
Update NuGet Feed Docs

### ISSUE THREAD

**Issue by yordan-mitev** [telerik/blazor-docs#444]
The Websites Team is working on upgrading the Telerik NuGet server to the latest API - v3. They plan to start the external beta testing with clients on September 1. For this purpose, the documentation must be updated to inform the clients about the new NuGet server. 

This is an excerpt from the announcement:
>**What’s Next**
> We are planning to release the new API in three phases:
>1.	Beta testing internally in the company (current phase) – You can test the new implementation by using it in test/demo projects and report issues. If any of our clients have problems with the current API you can send them to try the new API.
>2.	Beta testing externally – Announce the new API for our clients in the documentation with an invite to try the new API which is in beta version. We will need your assistance to update the documentation articles.
>3.	Officially switch to the new version – Update our documentation articles with the link to the new API and notification for all customers to switch to it if they have not already and let them know the old API is deprecated.
 
>You can find the 3.0 API at https://nuget.telerik.com/v3/index.json . The authentication method is the same as the current API, so you have to use the same credentials.
 
>**Desired Timeline**
>•	Phase 1 - ongoing, until 31 August
>•	Phase 2 - starts at 1 September, until 30 September
>•	Phase 3 - starts at 1 October, until 31 October

Depending on the results from the beta testing, we must plan the documentation updates that will be required for Phase 2 and 3.

My suggestion for Phase 2 is to insert an informational note in each document that mentions the Telerik NuGet URL. The note must be available on September 1.
>The improved Telerik NuGet v3 is now available for beta testing at https://nuget.telerik.com/v3/index.json. The new v3 API is faster, lighter, and reduces the number of requests from NuGet clients. You are welcome to try it.

Once the Websites team confirms that the beta testing is successful, we must plan and schedule the changes for Phase 3.

**Comment by yordan-mitev** [comment]
The [Installation ](https://docs.telerik.com/devtools/justmock/getting-started/installation-instructions)article must be updated to include the option to download the package from nuget.telerik.com.

**Comment by marin-bratanov** [comment]
I think we should write the final form of the articles - the new feed as the primary feed and so on, and the old one as a backup in a separate section. If we want to ask for feedback, I propose adding a note (in a template) for that that we can remove easily later. I don't think that the new feed should be a note, because there are many places that mention the current feed and adding a note everywhere will be confusing for people.

Let me know what you think, @yordan-mitev 

Btw, if nobody has time during the week, I can try to grab this next weekend.

**Comment by yordan-mitev** [comment]
I agree that writing the articles in their final form is the best approach. My concern is that the new v3 feed will be in beta testing till the end of September. The Websites team may need to make changes, extend the beta testing period, or face unexpected issues. That's why I suggested the cautious approach of using a note until the beta period is over.

With that said, every team decides how to change the affected docs independently so that the required iterations get reduced to a minimum and no additional support load gets generated.
